assignment-4/python/versionIncrement.py

```diff
@@ -0,0 +1,29 @@
+"""Holds function for incrementing and saving maya file"""
+
+import maya.cmds
+
+
+def incrementAndSave():
+    """
+    Increments the file version by one and saves.
+    Assumes the following file naming conventions:
+        asset.task.artist.version.ext
+    """
+    delimiter = "."
```

Comment:
interesting that you assigned "." to delimiter then just call back to it later. This is a cool way to do it.